_includes/inline-screenshot.html

```diff
@@ -1,10 +1,8 @@
 <div class="screenshot inline-screenshot pull-right">
 <a href="{{ include.source }}">
-<img src="{{ include.source }}" alt="screenshot"/>
+<img src="{{ include.source }}" {% if include.width %}style="width: {{include.width}}"{% endif %} alt="{% if include.caption %}{{ include.caption }}{% else %}screenshot{% endif %}"/>
```

Comment:
Can we get rid of this file and add a `include.inline` switch to `_includes/screenshot.html`?